llvm/lib/Target/SPIRV/SPIRVEmitIntrinsics.cpp

```diff
@@ -74,6 +74,10 @@ class SPIRVEmitIntrinsics
   void processInstrAfterVisit(Instruction *I);
   void insertAssignPtrTypeIntrs(Instruction *I);
   void insertAssignTypeIntrs(Instruction *I);
+
+  DenseSet<Instruction *> PtrCastInstrs;
+  void insertPtrCastInstr(Instruction *I);
+
```

Comment:
If we don't need PtrCastInstrs, we also don't need the empty line here  and at 77.